src/components/Signup/Signup.js

```diff
@@ -226,23 +234,13 @@ const Signup = function({ intl, dependencies: { dopplerLegacyClient, originResol
           </p>
         </footer>
       </article>
-      <section className="feature-panel">
-        <div
-          className="feature-panel--bg"
-          style={{
-            backgroundImage: `url('https://cdn.fromdoppler.com/doppler-ui-library/v2.5.0/img/violet-yellow.png')`,
-          }}
-        >
-          <article className="feature-content">
-            <h6>{_('feature_panel.email_automation')}</h6>
-            <h1>{_('feature_panel.email_automation_description')}</h1>
-            <p>{_('feature_panel.email_automation_remarks')}</p>
-          </article>
-          <figure className="content-img">
-            <img src={_('signup.image_path')} alt="Automation" />
-          </figure>
-        </div>
-      </section>
+      <Promotions type="signup" page={extractPage(location)} disabledSitesContent={true} />
+      <div
+        className="feature-panel--bg"
+        style={{
+          backgroundImage: `url('https://cdn.fromdoppler.com/doppler-ui-library/v2.5.0/img/violet-yellow.png')`,
```

Comment:
Is not the background part of _Promotions_ component?

Reply:
yes, problem with rebase. thanks